src/main/resources/templates/govuk/header.ftl

```diff
@@ -16,7 +16,7 @@
     <div class="govuk-header__logo">
       <a href="https://www.gov.uk" class="govuk-header__link govuk-header__link--homepage">
         <span class="govuk-header__logotype">
-          <svg role="presentation" focusable="false" class="govuk-header__logotype-crown" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 132 97" height="32" width="36">
+          <svg aria-hidden="true" focusable="false" class="govuk-header__logotype-crown" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 132 97" height="32" width="36">
```

Comment:
It was not in their [4.0.1 release notes](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0) but the [header markup](https://design-system.service.gov.uk/components/header/) has actually changed. The way the fallback image works is slightly different now.